README.md

```diff
@@ -1,24 +1,70 @@
-# README
+# "Hello world" project with rails-react gem
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+> Built using **_react-rails_** gem.
+## Built With
 
-Things you may want to cover:
+- Ruby
+- Ruby on Rails
+- ReactJS
+- Redux Toolkit
 
-* Ruby version
+## Getting Started
 
-* System dependencies
+- Getting a local copy is simple. Follow the steps below
 
-* Configuration
+### Install
 
-* Database creation
+- Ruby
+- Ruby on Rails
+- PostgresSQL
+- Rspec
 
-* Database initialization
+### Local Copy
 
-* How to run the test suite
+- Clone the project
 
-* Services (job queues, cache servers, search engines, etc.)
+```
+git clone https://github.com/akucintavalent/hello-rails-react.git
+cd hello-rails-react
+```
 
-* Deployment instructions
+### Setup
 
-* ...
+Install gems:
+
+```
+bundle install
+```
+
+Setup database:
+
+```
+rails db:create
+rails db:migrate
+```
+
+### Usage
+
+Start server:
+
+```
+rails server
+```
+
+Visit http://localhost:3000/ in your browser.
+
```

Comment:
- [ ] I would recommend mentioning the path where we can see the message. Currently, I can see nothing when I check on the root path.

![Screenshot from 2022-05-06 01-06-58](https://user-images.githubusercontent.com/26767383/167012640-4f8951c1-045a-4edb-8043-fd049e4eb2f5.png)


[OPTIONAL]

- [ ] Also, I would recommend adding the `seed` data to the `db/seeds.rb` so that user don't need to add the data by himself. Only `rails db:seed` would add all the data to the database.